intranet/apps/api/tests.py

```diff
@@ -184,22 +184,333 @@ def test_api_root(self):
     def test_api_eighth_block_list(self):
         self.make_token()
 
+        # Don't let blocks created in other tests contaminate these results
+        EighthBlock.objects.all().delete()
+
+        # List everything
         response = self.client.get(reverse("api_eighth_block_list"), HTTP_AUTHORIZATION=self.auth)
         self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.json()["results"], [])
 
         # Test a good date
         response = self.client.get(
```

Comment:
This function structure is used repeatedly and may be worthy of a refactor. It might be useful to create something like

```
def get_api_eighth_block_list(query):
    return self.client.get(reverse("api_eighth_block_list")+query, HTTP_AUTHORIZATION=self.auth)
```